flatdata-cpp/test/MultiVectorTest.cpp

```diff
@@ -251,6 +277,20 @@ TEST( MultiVectorTest, IterateOneTypeElementsRandomlyPlaced )
     ASSERT_TRUE( has_c2 );
 }
 
+TEST( MultiVectorTest, FlushingWhileBuilding )
+{
+    auto view = create_view_with_enough_items_to_flush( );
```

Comment:
This is not view. I would propose add another variable to avoid `view.second` below

Reply:
It is consistent with the other tests. Could make sense to change all of them at once at some point

Reply:
Would agree to the change of all of them at once.